wrapper.py

```diff
@@ -0,0 +1,1278 @@
+"""A thin pytorch / numpy compat layer.
+
+Things imported from here have numpy-compatible signatures but operate on
+pytorch tensors.
+"""
+import numpy as np
+import torch
+
+# Things to decide on (punt for now)
+#
+# 1. Q: What are the return types of wrapper functions: plain torch.Tensors or
+#       wrapper ndarrays.
+#    A: Tensors, apparently
```

Comment:
You should return torch.numpy.ndarray, how are you going to compose these functions into bigger ones otherwise?

Reply:
+1, I mentioned the same in gh-1.